### ISSUE TITLE
Problem getting started

### ISSUE THREAD

**Issue by DownInTheShop** [odwdinc/Python-SimConnect#18]
I apologize for wasting your time on something that's probably a non-issue. I"m a relative newbie to python, totally new to github, but pretty proficient at arduino. My plan is to build some generic instruments controlled via an arduino like altimeter, AOA, heading, etc and found this project that uses python to retrieve the simconnect data. When I run the local_example script I'm getting:
```
Traceback (most recent call last):
  File "C:/Users/Scott/PycharmProjects/SimConnect/main.py", line 1, in <module>
    from SimConnect import *
  File "C:\Users\Scott\PycharmProjects\SimConnect\venv\lib\site-packages\SimConnect\__init__.py", line 2, in <module>
    from .Entities import Plane
ModuleNotFoundError: No module named 'SimConnect.Entities'
```
Is this a problem on my end?

**Comment by odwdinc** [comment]
no its a bug, let me fix and get a new release out!

**Comment by odwdinc** [comment]
5a44caa17dcdf51fc900cef0aa35af9af0d1a2d4 resolves this bug see [SimConnect 0.3.5](https://pypi.org/project/SimConnect/0.3.5/)
